config/routes.rb

```diff
@@ -4,6 +4,7 @@
 
   resources :areas, only: %i[index] do
     resources :lines, only: %i[index]
+    get "/suggested_lines", to: "lines#suggested_lines"
```

Comment:
Should this be with the other GET's below?

Reply:
not sure, but i put it there because its nested in lines  and when i put it with the others it required a /:line_id